client/python/samples/vehicle_detection_predict_binary.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+from ovmsclient import make_grpc_client, make_grpc_predict_request
+from utils.utils import load_image, read_paths, get_single_metadata
+from utils.vehicle_utils import vehicle_postprocess
+
+parser = argparse.ArgumentParser(description='Make prediction using images in numpy format')
+parser.add_argument('--images_dir', required=True,
+                    help='Input images path.')
+parser.add_argument('--grpc_address', required=False, default='localhost',
+                    help='Specify url to grpc service. default:localhost')
+parser.add_argument('--grpc_port', required=False, default=9000,
+                    help='Specify port to grpc service. default: 9000')
+parser.add_argument('--model_name', default='vehicle-detection',
```

Comment:
What exact model do we target with this script?

Reply:
vehicle-detection model from here https://docs.openvinotoolkit.org/2019_R1/_vehicle_detection_adas_0002_description_vehicle_detection_adas_0002.html